src/app/global-search/global-search.component.ts

```diff
@@ -13,17 +13,29 @@ declare const google;
   styleUrls: ['./global-search.component.scss']
 })
 export class GlobalSearchComponent implements AfterViewInit {
+  /**
+   *
+   * Demo Mode
+   *
+   */
+
+  demoMode: boolean;
+  mockLocations: Array<any> = [];
+  displayLocations: boolean;
+  locationName: string;
+
+
   constructor(
     private router: Router,
     private searchService: SearchDataService,
     private locationService: LocationService,
     private ngZone: NgZone,
     private placesMock: PlacesMockService
   ) {
-    this.demoMode = environment.demoMode;
+    this.demoMode = environment.useMock;
```

Comment:
Please rename the component property to avoid confusion.

Reply:
Done.